packages/ra-core/src/controller/CreateController.tsx

```diff
@@ -50,7 +81,13 @@ import checkMinimumRequiredProps from './checkMinimumRequiredProps';
  *     );
  *     export default App;
  */
-export class CreateController extends Component {
+export class CreateControllerView extends Component<Props> {
```

Comment:
I don't agree with the renaming. In that particular case, it is not a view, it is a controller. If you're looking for a better name, try `CreateControllerCore` or `UnconnectedCreateController`.

Same for other renamings in ra-core controllers.